python/mmSolver/tools/removesolvernodes/ui/removesolvernodes_layout.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2020 John Smith.
```

Comment:
I have noticed you're using the name `John Smith`, is there a special reason?

I'm not 100% how copyrights work if you add a fake name.

I am happy for you to put your own name into the code (and in the `AUTHORS.txt` file).